### PR TITLE
Add the slp3 Release Note Content

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the slp3 release note content.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md [hunk 1]
```diff
@@ -22,9 +24,255 @@ However, you need to use the following commands instead of the above if you have
...
+- Support for executing stored procedures in the SQL connector
+- Azure Functions support
+
+### What's new in Ballerina Swan Lake Preview 3
+
+#### Language
+
+The language implementation is based on [Ballerina Language Specifications Draft 2020-06-18](https://ballerina.io/spec/lang/draft/v2020-06-18/).
+
+##### Extern method support
```
Comment (pubudu91) [inline downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md:39]: Use the latest content in the doc for this section

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md]: Done.

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md [hunk 2]
```diff
@@ -2,29 +2,279 @@
...
+chemistry: 95
+   };
+   Grade grades = student;
+}
+```
+By choosing the `Generate mapping function`code action, the following function will be added to the workspace.
+
+```ballerina
+function mapNameAndGradeToGrade(NameAndGrade nameAndGrade) returns Grade {
+// Some record fields might be missing in the AI based mapping.
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md:137]:
```suggestion
// Some record fields might be missing in the AI-based mapping.
```

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md [hunk 3]
```diff
@@ -2,29 +2,279 @@
...
+                    objectValue.getStringValue(new BmpStringValue("lname")));
+}
+```
+
 #### Standard Library
-- 
 
-#### Dev Tools
-- 
+##### JDBC
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md:69]: Is this only for `JDBC`. I assume for both `JDBC` and `MySQL` modules. Looping @niveathika

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md [hunk 4]
```diff
@@ -2,29 +2,279 @@
...
+    }
+
+    function getFullName() returns string = @java:Method {
+        class: "abc.Hello",
+    } external;
+};
+
+The Java method to which the `getFullName()` method is bound:
+
+```java
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md:60]: This is not rendering properly

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md (hunk 3 above)
Comment (niveathika) [reply downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md]: Yes, this is for both connectors, changed it as SQL connectors in 64ec347

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md [hunk 5]
```diff
@@ -2,29 +2,279 @@
...
+###### API change in `assertEquals` and `assertNotEquals` functions
+
+Deep value equality is supported only for `anydata`-typed values according to the language specification. The `assertEquals` function has been changed to accept only  `anydata`-typed values to reflect this behavior.
+
+###### Introduction of the `assertExactEquals` and `assertNotExactEquals` functions
+
+The `assertExactEquals` function compares two values to assert whether they refer to the same entity (i.e., they are exactly equal). 
+
+
+**Example:**
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md:173]: For some examples, we have this line. But not for all the examples. Can we be consistent on adding this?

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md (hunk 4 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md:60]: Fixed

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md (hunk 5 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md]: Fixed

File: downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md (hunk 4 above)
Comment (anupama-pathirage) [reply downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md]: Seems the fix has screwed up it further. Please check the rendering properly

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md]: Fixed.

**Pull request merged**